### PR TITLE
Worlds Docs: Translations German (Clique, BK Sudoku, OoT)

### PR DESCRIPTION
**Pull request by ZeitHeld**
~~I still dunno, if that is how you do a PR, but I hope it's good enough~~

## What is this fixing or adding?
3 Games have been translated into german (and even tried to keep the formatting rules). The Setup/Guide as well as the game info page(?) (since it has `en_` in front of it)

**Additions:**
- BK-Sudoku
- Clique
- OoT (With a screenshot)

Only one Screenshot has been added to Ocarina of Time (could be used universally), regarding the Patch Download for ROMS.

### CODE REVIEW

**Comment by BootsinSoots** [comment]
Thank you for your work! You did the PR right, good job. I advertised it a little so those who speak German can provide feedback (I do not speak it myself, so I'm not much help)

**Comment by Jarno458** [comment]
Well for BK sudoku with my limited German skiles, it seems alright, but you forgot something important for all 3 games
You need to add the new guides to the game its `WebWorld` often located in the `__init__.py`

**Comment by black-sliver** [comment]
Can we finally request a doc review from berserker? :D

**Comment by ZeitHeld** [comment]
> Well for BK sudoku with my limited German skiles, it seems alright, but you forgot something important for all 3 games You need to add the new guides to the game its `WebWorld` often located in the `__init__.py`

Ooh, I didn't really read that anywhere (tbh, going through the docs already is a bit obscure and stuff. Not quite obvious).
But I think I can look at it sooner or later if I can add it in by myself (shouldn't be too hard ig?)

**Comment by ZeitHeld** [comment]
~~what did that-~~

uuhhh, anyways, I added the lines of code into the init's and also added some "tranlation flexibility" into some others, that weren't made for multiple setups

Though, I think I made the changes into my "main" branch on accident- lemme fix that

**Comment by ZeitHeld** [comment]
Okay I think now it should be- (sorry about that)

**Comment by Jarno458** [comment]
Oke +1 for your enthusiasm, but you refactored a lot of init.py's, this is not idea as it put them in the scope of your PR and will require those maintainers to approve the PR in addition to the 3 maintainers of the guides you actually edited

**Comment by Jarno458** [comment]
While i admire your efforts to translate the game info page, i dont think there is currently any way to open those in a different language so that page will never be shown.. until a link to it is added somewhere

**Comment by ZeitHeld** [comment]
Yeah, the edits were not initially ideal, but made as it is really not a big change- And kost of them are ones I try to translate too

And the info pages, yeaah- I noticed that later on too, but thought it could be done due to them as starting with "en"
If it eventually gets added (possibly along with translated settings pages) they could be used

**Comment by black-sliver** [comment]
Please revert the changes to worlds that you didn't intend to modify. This will only create merge conflicts.

**Comment by ZeitHeld** [comment]
> Please revert the changes to worlds that you didn't intend to modify. This will only create merge conflicts.

Okay, I think that should all done now
Sorry about the issue with that

File: worlds/clique/docs/guide_de.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- Dein Zimmer aufräumen.
+- Die Wäsche machen.
+- Etwas Essen von einem X-Belieben Fast Food Restaruant holen.
+- Das tägliche Wordle machen.
+- ~~Deine Seele an **Phar** verkaufen.~~
+- Deine Hausaufgaben erledigen.
+- Deine Post abholen.
+
+
+~~Solltest du auf irgendwelche Probleme in diesem Spiel stoßen, solltest du aufkeinenfall nicht **thephar** auf~~
```
Comment (black-sliver) [inline worlds/clique/docs/guide_de.md:24]: auf keinen Fall
oder
keinenfalls

File: worlds/oot/docs/de_Ocarina of Time.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# The Legend of Zelda: Ocarina of Time
+
+## Wo ist die Seite für die Einstellungen?
+
+Die [Seite für die Spielereinstellungen dieses Spiels](../player-settings) enthält alle Optionen die man benötigt um
```
Comment (black-sliver) [inline worlds/oot/docs/de_Ocarina of Time.md:5]: neuer link
```suggestion
Die [Seite für die Spielereinstellungen dieses Spiels](../player-options) enthält alle Optionen die man benötigt um
```

File: worlds/oot/docs/de_Ocarina of Time.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+eine YAML-Datei zu konfigurieren und zu exportieren.
+
+## Was macht der Randomizer in diesem Spiel?
+
+Items welche der Spieler für gewöhnlich im Verlauf des Spiels erhalten würde, wurden umhergemischt. Die Logik bleit
```
Comment (black-sliver) [inline worlds/oot/docs/de_Ocarina of Time.md:10]:
```suggestion
Items, welche der Spieler für gewöhnlich im Verlauf des Spiels erhalten würde, wurden umhergemischt. Die Logik bleit
```

File: worlds/oot/docs/de_Ocarina of Time.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+bestehen, damit ist das Spiel immer durchspielbar. Doch weil die Items durch das ganze Spiel gemischt wurden, müssen
+ manche Bereiche früher bescuht werden, als man es in Vanilla tun würde.
+Eine Liste von implementierter Logik, die unoffensichtlich erscheinen kann, kann
+[hier (Englisch)](https://wiki.ootrandomizer.com/index.php?title=Logic) gefunden werden.
+
+## Welche Items und Bereiche werden gemischt?
+
+Alle ausrüstbare und sammelbare Gegenstände, sowie Munition können gemischt werden. Und alle Bereiche, die einen
+dieser Items enthalten könnten, haben (sehr wahrscheinlich) ihren Inhalt verändert. Goldene Skulltullas können ebenfalls
```
Comment (black-sliver) [inline worlds/oot/docs/de_Ocarina of Time.md:19]:
```suggestion
dieser Items enthalten könnten, haben (sehr wahrscheinlich) ihren Inhalt verändert. Goldene Skulltulas können ebenfalls
```

File: worlds/oot/docs/de_Ocarina of Time.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+dazugezählt werden, je nach Wunsch des Spielers.
+
+## Welche Items können in sich in der Welt eines anderen Spielers befinden?
+
+Jedes dieser Items, die gemicht werden können, können in einer Multiworld auch in der Welt eines anderen Spieler's
```
Comment (black-sliver) [inline worlds/oot/docs/de_Ocarina of Time.md:24]:
```suggestion
Jedes dieser Items, die gemicht werden können, können in einer Multiworld auch in der Welt eines anderen Spielers
```

File: worlds/oot/docs/de_Ocarina of Time.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+fallen. Es ist jedoch möglich ausgewählte Items auf deine eigene Welt zu beschränken.
+
+## Wie sieht ein Item einer anderen Welt in OoT aus?
+
+Items die zu einer anderen Welt gehören, werden repräsentiert durch Zelda's Brief.
```
Comment (black-sliver) [inline worlds/oot/docs/de_Ocarina of Time.md:29]:
```suggestion
Items, die zu einer anderen Welt gehören, werden repräsentiert durch Zelda's Brief.
```

File: worlds/oot/docs/de_Ocarina of Time.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+## Was passiert, wenn der Spieler ein Item erhält?
+
+Sobald der Spieler ein Item erhält, wird Link das Item über seinen Kopf halten und der ganzen Welt präsentieren.
+Gut für's Geschäft!
+
+## Einzigartige Lokale Befehle
+
+Die folgenden Befehle stehen nur im OoTClient, um mit Archipelago zu spielen, zur verfügung:
```
Comment (black-sliver) [inline worlds/oot/docs/de_Ocarina of Time.md:38]:
```suggestion
Die folgenden Befehle stehen nur im OoTClient, um mit Archipelago zu spielen, zur Verfügung:
```

File: worlds/oot/docs/setup_de.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+## WICHTIG
+
+Da wir BizHawk benutzen, gilt diese Anleitung nur für Windows und Linux.
+
+## Benötigte Software
+
+- BizHawk: [BizHawk Veröffentlichungen von TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+  - Version 2.3.1 und später werden unterstützt. Version 2.9 ist empfohlen.
+  - Detailierte Installtionsanweisungen für BizHawk können über den obrigen Link gefunden werden.
+  - Windows-Benutzer müssen die Prerequisiten (Darunter `Visual Studio 2019`) installiert haben. Diese können ebenfalls über
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:12]:
```suggestion
  - Windows-Benutzer müssen die Prerequisiten installiert haben. Diese können ebenfalls über
```
Es sollte sich dabei nur um die Runtime handeln, nicht um das (volle) Visual Studio. Ist jedenfalls irrelevant: Entweder die prereqs sind schon zufällig installiert, oder sie müssen laut link installiert werden.

File: worlds/oot/docs/setup_de.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+    den obrigen Link gefunden werden.
+- Der integrierte Archipelago-Client, welcher [hier](https://github.com/ArchipelagoMW/Archipelago/releases) installiert
+  werden kann. (wähle `Ocarina of Time Client` bei der Installation).
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:15]: Checkbox wurde entfernt und wird immer automatisch installiert.
```suggestion
  werden kann.
```

File: worlds/oot/docs/setup_de.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+- Eine `Ocarina of Time v1.0 US(?) ROM`. (Nicht aus Europa und keine Master-Quest oder Debug-Rom!)
+
+## Konfigurieren von BizHawk
+
+Sobald Bizhawk einmal Installiert wurde, öffne **EmuHawk** und ändere die folgenen Einsteluungen:
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:20]:
```suggestion
Sobald Bizhawk einmal installiert wurde, öffne **EmuHawk** und ändere die folgenen Einsteluungen:
```

File: worlds/oot/docs/setup_de.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+
+- (≤ 2.8) Gehe zu `Config > Customize`. Wechlse zu dem `Advanced`-Reiter, wechsle dann den `Lua Core` von "NLua+KopiLua" zu
+  `"Lua+LuaInterface"`. Starte danach EmuHawk neu. Dies ist zwingend notwendig, damit die Lua-Skripts (mit denen man sich mit dem Client verbindet) ordnungsgemäß funktionieren.
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:23]:
```suggestion
  `"Lua+LuaInterface"`. Starte danach EmuHawk neu. Dies ist zwingend notwendig, damit die Lua-Scripts, mit denen man sich mit dem Client verbindet, ordnungsgemäß funktionieren.
```
Scripts oder Skripte

File: worlds/oot/docs/setup_de.md [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+  **ANMERKUNG: Selbst wenn "Lua+LuaInterface" bereits ausgewählt ist, wechsle zwischen den beiden Optionen umher und**
+  **wähle es erneut aus. Neue Installationen oder Versionen von EmuHawk neigen dazu "Lua+LuaInterface" als die**
+  **Standard-Option anzuzeigen, aber laden dennoch "NLua+KopiLua", bis dieser Schritt getan ist.**
+- Unter `Config > Customize > Advanced`, gehe sicher dass der Haken bei `AutoSaveRAM` ausgeählt ist, und klicke dann
+  den 5s-Knopf. Dies verringert die Wahrscheinlichkeit den Speicherfrotschritt zu verlieren, sollte der Emulator mal
+  abstürzen.
+- **(Optional)** Unter `Config > Customize`, kannst du die Haken in den "Run in background"
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:30]: Glaub das Komma hier gehört nicht
```suggestion
- **(Optional)** Unter `Config > Customize` kannst du die Haken in den "Run in background"
```

File: worlds/oot/docs/setup_de.md [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+  (Laufe weiter im Hintergrund) und "Accept background input" (akzeptiere Tastendruck im Hintergrund) Kästchen setzen.
+  Dies erlaubt dir das Spiel im Hintergrund weiter zu spielen, selbst wenn ein anderes Fenster aktiv ist. (Nützlich bei
+  mehreren oder eher großen Bildschrimen/Monitoren.)
+- Unter `Config > Hotkeys`, sind viele Hotkeys, die mit oft genuten Tasten belegt worden sind. Es wird empfohlen die
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:34]: wie oben
```suggestion
- Unter `Config > Hotkeys` sind viele Hotkeys, die mit oft genuten Tasten belegt worden sind. Es wird empfohlen die
```

File: worlds/oot/docs/setup_de.md [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+  meisten (oder alle) Hotkeys zu deaktivieren. Dies kann schnell getan werden mit `Esc`.
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:35]:
```suggestion
  meisten (oder alle) Hotkeys zu deaktivieren. Dies kann schnell mit `Esc` erledigt werden.
```

File: worlds/oot/docs/setup_de.md [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+- Wird mit einem Kontroller gespielt, bei der Tastenbelegung (bei einem Laufendem Spiel, unter
+  `Config > Controllers...`), deaktiviere "P1 A Up", "P1 A Down", "P1 A Left", and "P1 A Right" und gehe stattdessen in
+  den Reiter `Analog Controls` um den Stick zu belegen, da sonst Probleme auftreten bei dem Zielen (mit dem Bogen oder
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:38]:
```suggestion
  den Reiter `Analog Controls` um den Stick zu belegen, da sonst Probleme beim Zielen auftreten (mit dem Bogen oder
```

File: worlds/oot/docs/setup_de.md [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+  ähnliches). Y-Axis ist für Oben und Unten, und die X-Axis ist für Links und Rechts.
+- Unter `N64` setze einen Haken bei "Use Expansion Slot" (Benutze Erweiterungs-Slot). Dies wird benötigt damit
+  savestates/schnellspeichern funktioniert. (Das N64-Menü taucht nur auf **nach** dem laden einer N64-ROM.)
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:41]:
```suggestion
  savestates/schnellspeichern funktioniert. (Das N64-Menü taucht nur **nach** dem laden einer N64-ROM auf.)
```

File: worlds/oot/docs/setup_de.md [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+sich der Emulator befindet, und wähle dann `EmuHawk.exe` **(NICHT "DiscoHawk.exe"!)** aus.
+
+Eine Alternative BizHawk Setup Anleitung (auf Englisch), sowie weitere Hilfe bei Problemen kann
+[hier](https://wiki.ootrandomizer.com/index.php?title=Bizhawk) gefunden werden.
+
+## Erstelle eine YAML-Datei
+
+### Was ist eine YAML-Datei und Warum brauch ich eine?
+
+Eine YAML-Datie enthält einen Satz an einstellbaren Optionen, welche Informationen den Zufallsgenerator liefert wie
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:56]: Vielleicht einfacher so?
```suggestion
Eine YAML-Datie enthält einen Satz an einstellbaren Optionen, die dem Generator mitteilen, wie
```

File: worlds/oot/docs/setup_de.md [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+dein Spiel generiert werden soll. In einer Multiworld stellt jeder Spieler eine eigene YAML-Datei zur Verfügung. Dies
+erlaubt jeden Spieler eine personalisierte Erfahrung nach derem Geschmack. Damit kann auch jeder Spieler in einer
+Multiworld (des gleichen Spiels) völlig unterschiedliche Einstellungen haben.
+
+Für weitere Informationen besuche die Anleitung zum erstellen einer gewöhnlichen
+YAML-Datei: [Gewhönliche Multiworld Setup Antelitung](/tutorial/Archipelago/setup/de)
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:62]: "Gewöhnlich" ist nicht gut übersetzt. "Allgemein" wäre richtig. /de-Link gibt es noch nicht, oder?
```suggestion
Für weitere Informationen, besuche die allgemeine Anleitung zum Erstellen einer
YAML-Datei: [Archipelago Setup Anleitung](/tutorial/Archipelago/setup/en)
```

File: worlds/oot/docs/setup_de.md [hunk 19]
```diff
@@ -0,0 +1,108 @@
...
+
+### Woher bekomme ich eine YAML-Datei?
+
+Die Seite für die Spielereinstellungen auf dieser Website erlaubte es dir deine persönlichen Einstellungen nach
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:66]: Tippfehler
```suggestion
Die Seite für die Spielereinstellungen auf dieser Website erlaubt es dir deine persönlichen Einstellungen nach
```

File: worlds/oot/docs/setup_de.md [hunk 20]
```diff
@@ -0,0 +1,108 @@
...
+vorlieben zu konfigurieren und eine YAML-Datei zu exportieren.
+Seite für die Spielereinstellungen:
+[Seite für die Spielereinstellungen von Ocarina of Time](/games/Ocarina%20of%20Time/player-settings)
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:69]: neuer link
```suggestion
[Seite für die Spielereinstellungen von Ocarina of Time](/games/Ocarina%20of%20Time/player-options)
```

File: worlds/oot/docs/setup_de.md [hunk 21]
```diff
@@ -0,0 +1,108 @@
...
+
+### Überprüfen deiner YAML-Datei
+
+Wenn du deine YAML-Datei überprüfen möchtest, um sicher zu gehen dass sie funktioniert, kannst du dies auf der
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:73]:
```suggestion
Wenn du deine YAML-Datei überprüfen möchtest, um sicher zu gehen, dass sie funktioniert, kannst du dies auf der
```

File: worlds/oot/docs/setup_de.md [hunk 22]
```diff
@@ -0,0 +1,108 @@
...
+YAML-Überprüfungsseite tun.
+YAML-Überprüfungsseite: [YAML-Überprüfungsseite](/mysterycheck)
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:75]: neuer link
```suggestion
YAML-Überprüfungsseite: [YAML-Überprüfungsseite](/check)
```

File: worlds/oot/docs/setup_de.md [hunk 23]
```diff
@@ -0,0 +1,108 @@
...
+
+## Beitreten einer Multiworld
+
+### Erhalte deinen OoT-Patch
+
+(Der folgende Prozess ist bei den meisten ROM-basierenden Spielen sehr ähnlich.)
+
+Wenn du einer Multiworld beitrittst, wirst du gefordert eine YAML-Datei bei dem Host abzugeben. Ist dies getan, wird
+dich der Host der Multiworld mit (in der Regel) einem Link, der zu einem Raum führt. Dort sind alle Teilnehmenden
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:84]: Zweiter Satz hier ist nicht vollständig?

File: worlds/oot/docs/setup_de.md [hunk 24]
```diff
@@ -0,0 +1,108 @@
...
+`Download APZ5 File...`.
+![Screenshot of a Multiworld Room with an Ocarina of Time Player](/static/generated/docs/Ocarina%20of%20Time/MultiWorld-room_oot.png)
+
+Führe die `.apz5`-Datei mit einem Doppelklick aus, um deinen Ocarina Of Time-Client zu starten, sowie das patchen
+deiner ROM. Ist dieser Prozess fertig (kann etwas dauern), startet sich der Client und der Emulator automatisch
+(sofern das "Öffnen mit..." ausgewählt wurde).
+
+### Verbinde zum Multiserver
+
+Sind einmal der Client und der Emulator gestartet, müssen sie nur noch mit einander Verbunden werden. Gehe dazu in
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:95]:
```suggestion
Sind einmal der Client und der Emulator gestartet, müssen sie nur noch miteinander verbunden werden. Gehe dazu in
```

File: worlds/oot/docs/setup_de.md [hunk 25]
```diff
@@ -0,0 +1,108 @@
...
+deinen Archipelago-Ordner, dann zu `data/lua`, und füge das `connector_oot.lua` Script per Drag&Drop (ziehen und
+fallen lassen) auf das EmuHawk-Fenster. (Alternativ kannst du die Lua-Konsole manuell öffnen, gehe dazu auf
+`Script > Open Script` und durchsuche die Ordner nach `data/lua/connector_oot.lua`.)
+
+Um den Client mit dem Multiserver zu verbinden, füge einfach `<Adresse>:<Port>` ein auf das Textfeld ganz oben im
+Client und drücke Enter oder "Connect" (verbinden). Wird Ein Passwort benötigt musst du es danach unten in das
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:101]:
```suggestion
Um den Client mit dem Multiserver zu verbinden, füge einfach `<Adresse>:<Port>` in das Textfeld ganz oben im
Client ein und drücke Enter oder "Connect" (verbinden). Wird ein Passwort benötigt, musst du es danach unten in das
```

**Review by black-sliver — COMMENTED** [review]
I feel bad for the amount of suggestions.
German Komma be German Komma. Probably still missed some.

**Comment by ZeitHeld** [comment]
> I feel bad for the amount of suggestions. German Komma be German Komma. Probably still missed some.

It's a diffciult langunge with it's forms of politeness, but I think/hope that after this weird sentence everything should be good.
Unless there is another issue found by someone or someone else
Don't worry about the amount. I shouldn't have done them at a late time lmao

**Comment by black-sliver** [comment]
> but I think/hope that after this weird sentence everything should be good.

This is still to be done, right? (I see two conversations that are not marked as resolved yet.)

File: worlds/bk_sudoku/docs/de_Sudoku.md [hunk 26]
```diff
@@ -0,0 +1,21 @@
...
+beliebigen Multiworld verbinden kann. Einmal verbunden kannst du ein 9x9 Sudoku spielen um einen zufälligen Hinweis
+für dein Spiel zu erhalten. Es ist zwar langsam, aber es gibt dir etwas zu tun, solltest du mal nicht in der Lage sein
+weitere Check zu erreichen.
+(Wer mag kann auch einfach so Sudoku spielen, man muss nicht mit einer Multiworld verbunden sein, um ein Sudoku zu
+spielen/generieren.)
+
+## Wie werden Hinweise freigeschalten?
+
+Nach dem Lösen eines Sudokus wird das Spiel einen zufällig ausgewählten Hinweis freigeben für einen noch nicht
+gefundenen Check in dem Slot mit dem du verbunden bist.
```
[the remark this answers is not in the document]
Comment (ZeitHeld) [reply worlds/bk_sudoku/docs/de_Sudoku.md]: ich habe das etwas kombiniert, und denke das klingt jetzt auch etwas besser

**Comment by ZeitHeld** [comment]
Okay, I think that should be done now?
At least it seems to be resolved

File: worlds/bk_sudoku/__init__.py [hunk 27]
```diff
@@ -7,16 +7,25 @@
...
-    ]
+    )
+    setup_de = Tutorial(
+            tutorial_name='Setup Anleitung',
+            description='Eine Anleitung um BK-Sudoku zu spielen',
+            language='Deutsch',
+            file_name='setup_de.md',
+            link='setup/de',
+            authors=['Held_der_Zeit']
+    )
```
Comment (black-sliver) [inline worlds/bk_sudoku/__init__.py:26]: Hm, this still has wrong indentation
```python
    setup_en = Tutorial(
        tutorial_name='Setup Guide',
        description='A guide to playing BK Sudoku',
        language='English',
        file_name='setup_en.md',
        link='setup/en',
        authors=['Jarno']
    )
    setup_de = Tutorial(
        tutorial_name='Setup Anleitung',
        description='Eine Anleitung um BK-Sudoku zu spielen',
        language='Deutsch',
        file_name='setup_de.md',
        link='setup/de',
        authors=['Held_der_Zeit']
    )
```
would be correct.

File: worlds/oot/docs/setup_de.md [hunk 28]
```diff
@@ -0,0 +1,108 @@
...
+YAML-Überprüfungsseite: [YAML-Überprüfungsseite](/check)
+
+## Beitreten einer Multiworld
+
+### Erhalte deinen OoT-Patch
+
+(Der folgende Prozess ist bei den meisten ROM-basierenden Spielen sehr ähnlich.)
+
+Wenn du einer Multiworld beitrittst, wirst du gefordert eine YAML-Datei bei dem Host abzugeben. Ist dies getan,
+erhälst du (in der Regel) einem Link, von dem Host der Multiworld. Dieser führt dich zu einem Raum in dem alle
```
Comment (black-sliver) [inline worlds/oot/docs/setup_de.md:84]: Glaub so:
```suggestion
erhälst du (in der Regel) einen Link vom Host der Multiworld. Dieser führt dich zu einem Raum, in dem alle
```

**Review by black-sliver — APPROVED** [review]
Looks good now. I DM'd phar and espeon to make sure they don't have any objections.

**Pull request merged**